src/main/java/micdoodle8/mods/galacticraft/api/spaceprojects/GCGlobalVariableStorage.java

```diff
@@ -0,0 +1,12 @@
+package micdoodle8.mods.galacticraft.api.spaceprojects;
+
+import java.util.HashMap;
+
+public abstract class GCGlobalVariableStorage {
+    // Maps user IDs to usernames and vice versa. Seems redundant but this makes accessing this
+    // easier in certain locations (like gt commands).
+    public static HashMap<String, String> GlobalSpaceProjectName = new HashMap<>(100, 0.9f);
```

Comment:
Sounds like you'd want a single BiMap instead of two Maps here

Reply:
Will try

Reply:
How could I do it wit ha single map? cause if i Use a single I don't think i can even connect two users together in the same team